droidlet/memory/robot/loco_memory.py

```diff
@@ -94,3 +94,8 @@ def get_detected_objects_tagged(self, *tags) -> List["DetectedObjectNode"]:
     def add_dance(self, dance_fn, name=None, tags=[]):
         # a dance is movement determined as a sequence of steps, rather than by its destination
         return DanceNode.create(self, dance_fn, name=name, tags=tags)
+
+    def clear(self, objects):
```

Comment:
would be better to just overload .forget I think

Reply:
.forget takes one memid where as this takes a list of them. Hence didn't / couldn't overload it.